pages/_document.tsx

```diff
@@ -6,6 +6,7 @@ export default function Document() {
         <Html>
             <Head>
                 {/* Favicon  */}
+                <link rel="shortcut icon" href="../public/assets/images/favicon.ico" />
```

Comment:
why need this change? Is is really matter to the search engine?  
Maybe u send me the reference for this change.

Reply:
https://developers.google.com/search/docs/appearance/favicon-in-search?hl=zh-tw
it has a method that <link>is used without type. If the icon doesn't change this week, i want to try if this works